src/test/java/org/takes/facets/auth/PsBasicTest.java

```diff
@@ -209,4 +211,63 @@ private String generateAuthenticateHead(
         );
         return String.format(AUTH_BASIC, encoded);
     }
+
+    /**
+     * Test of {@link PsBasic.Default}.
+     * @author lcozzani (lautaromail@gmail.com)
+     * @version $Id$
+     * @since 0.22
+     */
+    public static final class Default {
```

Comment:
@lautarobock Let's avoid declaring nested test classes. It's fine to just test `PsBasic.Default` within `PsBasicTest`.